dtfs-central-api/src/v1/swagger-definitions/utilisation-reports/utilisation-reports-reconciliation-summary.js

```diff
@@ -24,6 +24,8 @@
  *       dateUploaded:
  *         type: string
  *         example: 2021-01-01T00:00:00.000Z
+ *       totalFacilitiesReported:
+ *         type: number
```

Comment:
I think the optional fields on the `UtilisationReportReconciliationSummaryItem` type need to be updated in these docs, but I'm not sure what they should be updated to. I think `required` tag is only for request values, and `nullable: true` doesn't exactly work as the field wouldn't be `null` and would instead just not be present. What do you think?

Reply:
So I think the red stars indicate required, so if anything we might need to be putting required on more fields?
<img width="936" alt="Screenshot 2024-07-08 at 15 44 58" src="https://github.com/UK-Export-Finance/dtfs2/assets/67907735/9cef4971-7d96-40cc-99c9-7555960bfafd">

Reply:
Interesting, although it seems strange to me to tell a user that the field we are returning to them is "required". I guess many of our swagger docs aren't accurate with respect to this optional behaviour anyway, so let's just leave this for now